### ISSUE TITLE
Crashes on App startup

### ISSUE THREAD

**Issue by PanterSoft** [macmade/Hot#78]
Hello,

The app is crashing when being started. it quickly flashes in the menu bar and then it is gone.

In the system Monitor the app is still there but it cannot be accessed again. Only solve is to close it out in system monitor and restart the app.

I am using an M1 MacBook Pro with the latest macOS Sonoma 14.0 (23A344)

Can I somehow pull an Log which would help ?

**Comment by PanterSoft** [comment]
It fixed itself somehow after multiple reboots

**Comment by PanterSoft** [comment]
Issue is back again. and also Temperature reading is showing 51 degrees the whole time (Probably Program is hung up) but still being displayed in Menu Bar

**Comment by macmade** [comment]
Can you sample the process from Activity Monitor?

**Comment by PanterSoft** [comment]
Hello @macmade

Yes I can. Here is the Sample

[Hot_Sample.txt](https://github.com/macmade/Hot/files/12827862/Hot_Sample.txt)

**Comment by xaviergmail** [comment]
This seems to happen to me on a 2019 MBP just recently after upgrading to Sonoma. Hot was running just a few minutes ago and the scheduler must've dropped down below 70% because my laptop was nearly unusable so I went to check, but the app had crashed. I tried re-opening the app but it kept crashing until xcode finished installing and the app started working again 🤷 

Wish I could give more input but that's been my experience with it today

**Comment by PanterSoft** [comment]
Hello @xaviergmail,

For me the Issue is not Load sensitive I tried restarting the app while my Macbook is in Idle with no success the issue still persists.

But I got new information. in The Tab Temperature Sensors all the temperature sensors are working. (The Temperatures change while in use...) but the CPU temp displayed in the menu bar is still showing 51 °C all the time no matter if the MacBook is running full load or idle.
